### ISSUE TITLE
Train and Test split for NQ

### ISSUE THREAD

**Issue by VityaVitalich** [voidism/Lookback-Lens#8]
Dear maintainers,
I am trying to take your work as an example for my research and would like to adopt the split of train and test for NQ.
In the paper I see that there is 2655 samples for test and 2499 for train. However, only 1 dataset for NQ is present in this repo which size is 2655, which is used for both training and evaluation.
Am I missing something?

**Comment by voidism** [comment]
Hi,

Sorry for the late reply! And sorry that I haven't check the issues of this repo for a long time. The 2499 training examples was sampling from the Natural Questions training set, they are different from the test set. You can download the file here: https://drive.google.com/file/d/1YU_a3KH6dG_d1I6w5Et2aZl34n-D5qDx/view?usp=sharing

Hope it helps!
